app/gui2/src/stores/project/executionContext.ts

```diff
@@ -29,6 +29,8 @@ import { exponentialBackoff } from 'shared/util/net'
 import type { ExternalId } from 'shared/yjsModel'
 import { reactive } from 'vue'
 
+const DEFAULT_ENVIRONMENT: ExecutionEnvironment = 'Design'
```

Comment:
Is there a place where we should look to keep this synced with Java?

Reply:
I'm not aware of any "GUI-Engine common constants" place in new GUI. There was something of sort in the old GUI, but didn't feel practical.